src/index.tsx

```diff
@@ -93,10 +105,17 @@ export async function init(config: IClientConfig): Promise<EppoClient> {
       pollingIntervalMs: config.pollingIntervalMs ?? undefined,
     };
 
-    EppoReactNativeClient.instance.setLogger(config.assignmentLogger);
+    EppoReactNativeClient.instance.setAssignmentLogger(config.assignmentLogger);
+
+    const storageKeySuffix = buildStorageKeySuffix(config.apiKey);
+    const assignmentCache = assignmentCacheFactory({
+      storageKeySuffix,
+    });
+    if (assignmentCache instanceof HybridAssignmentCache) {
+      await assignmentCache.init();
+    }
+    EppoReactNativeClient.instance.useCustomAssignmentCache(assignmentCache);
 
-    // by default use non-expiring assignment cache.
-    EppoReactNativeClient.instance.useNonExpiringInMemoryAssignmentCache();
```

Comment:
We have been using an in-memory assignment cache. Using a hybrid assignment cache would make this more consistent with what happens in the async init methods in the js-client-sdk